playwright/components/pill/index.ts

```diff
@@ -0,0 +1,7 @@
+import { Page } from "playwright-core";
+import PILL_CLOSE_ICON from "./locators";
+
+// component preview locators
+const pillCloseIcon = (page: Page) => page.locator(PILL_CLOSE_ICON);
```

Comment:
**comment** (non-blocking): looks like this isn't used in your code, but yeah, I agree it should just be left in. Hopefully the individual who goes to do the pill refactor realises the actual pill locator is in the general one ;)

Reply:
Yeah same reason as the `split-button`. Converted it before I started the actual refactoring of the component file. hence, I just left as is as we'll need it in the future anyway.